README.md

```diff
@@ -0,0 +1,51 @@
+[![Build Status](https://travis-ci.org/gnosis/dex-price-estimator.svg?branch=master)](https://travis-ci.org/gnosis/dex-price-estimator)
+# Dex Price Estimator
+A service estimating feasible sell prices for Gnosis Protocol.
+
+## API
+
+The service exposes two endpoints:
+
+- *GET /ordberbook?base=\<tokenID>&quote=\<tokenID>*: returns the transitive orderbook (containing bids and asks) for the given base and quote token
+- *GET /price?base=\<tokenID>&quote=\<tokenID>&sell=\<AMOUNT>*: estimates the price a user would have to bid to sell the amount of base token for the given quote token
```

Comment:
What is expected to happen when, for example, there is no price estimate? I have tried to get an estimate selling 100 USDC for ETH and got nothing back.

```
https://price-estimate-mainnet.dev.gnosisdev.com/price?base=4&quote=1&sell=100000000000000000000
```

Reply:
If there is no price estimate you get an empty response. Since USDC has only 6 decimals you asked for a price to sell 100000000 USDC, for which we indeed don't have enough liquidity (yet 😉) 

If you query a more reasonable amount, e.g. https://price-estimate-mainnet.dev.gnosisdev.com/price?base=4&quote=1&sell=1000000 you get a price (it does however not account for decimals)

Reply:
Nice thanks!

> it does however not account for decimals

Should we then just return integer price estimates?